resource/interface/client/skillcheck.lua

```diff
@@ -22,6 +22,20 @@ function lib.skillCheck(difficulty, inputs)
     return Citizen.Await(skillcheck)
 end
 
+function lib.cancelSkillCheck()
+    if not skillcheck then
+        error('No skillCheck is active')
```

Comment:
Instead of making an error, I'd advise to just return false if it fails and true if it succeeded

Reply:
> Instead of making an error, I'd advise to just return false if it fails and true if it succeeded

They did same for progress bar cancel I agree with what you saying but I am just following ox standards